Document the cleanup operator (#609)
astronomer/astro-sdk: docs/astro/sql/operators/cleanup.rst

======================================
cleanup operator
======================================

When to Use cleanup operator
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> Cleanup operator is used to clean :ref:`temp_table` created. It monitors the status of all the other tasks and execute at the end to clear all the :ref:`temp_table` created in a dag run. It is recommended to add cleanup operator in every DAG that uses AstroSDK.

Reviewer: ```suggestion
Cleanup operator is used to clean :ref:`temp_table` created. It monitors the status of all the other tasks and execute at the end to clear all the :ref:`temp_table` created in a dag run. It is recommended to add ``cleanup`` operator in every DAG that uses the Astro SDK.
```

======================================
cleanup operator
======================================

.. _cleanup_operator:

> When to Use cleanup operator

Reviewer: ```suggestion
When to use the ``cleanup`` operator
```

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> Cleanup operator is used to clean temporary tables(:ref:`table`) created. It monitors the status of all the other tasks and execute at the end to clear all the temporary tables(:ref:`table`) created in a dag run. It is recommended to add ``cleanup`` operator in every DAG that uses the Astro SDK.

Reviewer: ```suggestion
The ``cleanup`` operator is used to clean temporary tables(:ref:`table`). It monitors the status of all the tasks within a DAG and deletes the created temporary tables(:ref:`table`) by the end of the DAG run. It is recommended to add ``cleanup`` operator in every DAG that uses the Astro SDK.
```

It is probably worth mentioning that the cleanup operator can be used to clean all or a subset of temporary tables, depending on how it is used, and have examples of how it can be used to clean up a subset. Something like:
```
cleanup(temp_table, temp_table)
```
